src/ts/components/audioqualityselectbox.ts

```diff
@@ -52,6 +52,12 @@ export class AudioQualitySelectBox extends SelectBox {
     player.on(player.exports.PlayerEvent.PeriodSwitched, updateAudioQualities);
     // Update quality selection when quality is changed (from outside)
     player.on(player.exports.PlayerEvent.AudioQualityChanged, selectCurrentAudioQuality);
+    if (player.exports.PlayerEvent.hasOwnProperty('AudioQualityAdded')) {
```

Comment:
Please write these checks similar to their usage below (`if ((player.exports.PlayerEvent as any).AudioQualityAdded) { ...`) to avoid strings which are easily missed in searches and refactorings/renamings.